packages/cli/src/commands/index.ts

```diff
@@ -139,6 +139,32 @@ export function registerScriptCommand(program: Command) {
     )
     .action(lazy(() => import('./export-dynamic-plugin').then(m => m.command)));
 
+  command
+    .command('package-dynamic-plugins')
+    .description(
+      'Package up exported dynamic plugins as container image for deployment',
+    )
+    .option(
+      '--force-export',
+      'Regenerate the dist-dynamic folder for each plugin even if it already exists',
+    )
+    .option(
+      '--preserve-temp-dir',
+      'Leave the temporary staging directory on the filesystem instead of deleting it',
+    )
+    .requiredOption(
+      '-t, --tag <tag>',
+      'Tag name to use when building the plugin registry image',
+    )
+    .option(
+      '--container-tool <tool>',
+      'The container tool to use for building the image (podman or docker, podman is the default)',
```

Comment:
Would it be possible to limit it for the supported tools only and avoid user to try something that we do not expect.